core/templates/dev/head/editor/ExplorationFeedback.js

```diff
@@ -146,6 +146,16 @@ oppia.controller('ExplorationFeedback', [
     });
   };
 
+  $scope.isSendDisabled = function(updatedStatus) {
+    if ($scope.messageSendingInProgress == true) {
+      return true;
+    }
+    else {
```

Comment:
nit: move "else {" to previous line. In general follow the Google Javascript style guide: http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=Code_formatting#Code_formatting